src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas1/Cas1AssessmentEmailService.kt

```diff
@@ -50,6 +50,25 @@ class Cas1AssessmentEmailService(
     }
   }
 
+  fun appealedAssessmentAllocated(allocatedUser: UserEntity, assessmentId: UUID, crn: String) {
+    allocatedUser.email?.let { email ->
+      emailNotificationService.sendEmail(
+        recipientEmailAddress = email,
+        templateId = notifyConfig.templates.appealedAssessmentAllocated,
+        personalisation = mapOf(
+          "name" to allocatedUser.name,
+          "assessmentUrl" to assessmentUrlTemplate.resolve("id", assessmentId.toString()),
+          "crn" to crn,
+          "deadlineCopy" to mapOf(
+            "name" to allocatedUser.name,
+            "assessmentUrl" to assessmentUrlTemplate.resolve("id", assessmentId.toString()),
+            "crn" to crn,
```

Comment:
this doesn't appear to align with the template in notify. Is deadline copy meant to be a map?

Reply:
Whoops!